src/components v2/Molecules/Forms/InputText/InputText.vue

```diff
@@ -22,6 +22,7 @@
       :disabled="disabled"
       :value="value"
       :type="type"
+      v-on="$listeners"
```

Comment:
I have my reservations on this. We should be explicit about bindings. This may mean that we get events happening that we do not intend.

Can you bubble up only the blur and focus properties?

That way we know what events we can and what events we can't pass up when looking at the component.

```
@blur="$emit('blur', $event)"
@focus="$emit('focus', $event)"
```

That way we know what this component does and we don't have to open others to understand what its doing.

Reply:
sure thing

this looks like a professional did it